shared-data/labware/definitions/2/greiner_384_wellplate_240ul/1.json

```
@@ -0,0 +1,5203 @@
+        "F15",
+        "G15",
+        "H15",
+        "I15",
+        "J15",
+        "K15",
+        "L15",
+        "M15",
+        "N15",
+        "O15",
+        "P15",
+        "A16",
+        "B16",
+        "C16",
+        "D16",
+        "E16",
+        "F16",
+        "G16",
+        "H16",
+        "I16",
+        "J16",
+        "K16",
+        "L16",
+        "M16",
+        "N16",
+        "O16",
+        "P16",
+        "A17",
+        "B17",
+        "C17",
+        "D17",
+        "E17",
+        "F17",
+        "G17",
+        "H17",
+        "I17",
+        "J17",
+        "K17",
+        "L17",
+        "M17",
+        "N17",
+        "O17",
+        "P17",
+        "A18",
+        "B18",
+        "C18",
+        "D18",
+        "E18",
+        "F18",
+        "G18",
+        "H18",
+        "I18",
+        "J18",
+        "K18",
+        "L18",
+        "M18",
+        "N18",
+        "O18",
+        "P18",
+        "A19",
+        "B19",
+        "C19",
+        "D19",
+        "E19",
+        "F19",
+        "G19",
+        "H19",
+        "I19",
+        "J19",
+        "K19",
+        "L19",
+        "M19",
+        "N19",
+        "O19",
+        "P19",
+        "A20",
+        "B20",
+        "C20",
+        "D20",
+        "E20",
+        "F20",
+        "G20",
+        "H20",
+        "I20",
+        "J20",
+        "K20",
+        "L20",
+        "M20",
+        "N20",
+        "O20",
+        "P20",
+        "A21",
+        "B21",
+        "C21",
+        "D21",
+        "E21",
+        "F21",
+        "G21",
+        "H21",
+        "I21",
+        "J21",
+        "K21",
+        "L21",
+        "M21",
+        "N21",
+        "O21",
+        "P21",
+        "A22",
+        "B22",
+        "C22",
+        "D22",
+        "E22",
+        "F22",
+        "G22",
+        "H22",
+        "I22",
+        "J22",
+        "K22",
+        "L22",
+        "M22",
+        "N22",
+        "O22",
+        "P22",
+        "A23",
+        "B23",
+        "C23",
+        "D23",
+        "E23",
+        "F23",
+        "G23",
+        "H23",
+        "I23",
+        "J23",
+        "K23",
+        "L23",
+        "M23",
+        "N23",
+        "O23",
+        "P23",
+        "A24",
+        "B24",
+        "C24",
+        "D24",
+        "E24",
+        "F24",
+        "G24",
+        "H24",
+        "I24",
+        "J24",
+        "K24",
+        "L24",
+        "M24",
+        "N24",
+        "O24",
+        "P24"
+      ],
+      "metadata": {
+        "wellBottomShape": "v"
+      }
+    }
+  ],
+  "parameters": {
+    "format": "384Standard",
+    "quirks": [],
+    "isTiprack": false,
+    "loadName": "greiner_384_wellplate_240ul",
+    "isMagneticModuleCompatible": false
+  },
+  "namespace": "opentrons",
+  "version": 1,
+  "schemaVersion": 2,
+  "cornerOffsetFromSlot": {
+    "x": 0,
+    "y": 0,
+    "z": 0
+  },
+  "stackingOffsetWithLabware": {},
+  "stackingOffsetWithModule": {},
+  "allowedRoles": [],
```

`allowedRoles` is underdocumented so I'm not sure, but I suspect you want this to be:

```suggestion
  "allowedRoles": ["labware"],
```